jb/feed_error.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef jb_feed_error_hpp
+#define jb_feed_error_hpp
+
+#include <stdexcept>
+
+namespace jb {
+
+/**
+ * Communicate feed error exceptions.
+ */
+class feed_error : public std::runtime_error {
+ public:
+  feed_error(std::string const& msg)
+      : std::runtime_error("Feed error exception caught -> " + msg) {}
```

Comment:
This is not the point where the exception is caught, is it?  I think just "msg", or if you want: "feed error: " + msg